Test/ParallelPrimitives/RadixSortConfigs.h

```diff
@@ -30,4 +29,6 @@ constexpr auto SINGLE_SORT_WG_SIZE{ 128 };
 // scan configs
 constexpr auto SCAN_WG_SIZE{ BIN_SIZE };
 
+constexpr auto KEY_VALUE_PAIR_ENABLED{ true };
```

Comment:
Change this variable to try different versions (key-only vs key-value pair).  Need to recompile the kernels.